items.json

```diff
@@ -20930,11 +20944,13 @@
     "name": "Broad Blade Javelin",

```

Comment:
This is a huge nerf. Please change the id of the weapon to refund it. add v2 at the end

Reply:
please close PR request, stuff is outdated at this point